src/EditPokemon/EditPokemon.js

```diff
@@ -35,18 +37,24 @@ export default function EditPokemon(props) {
                     type="text"
                     defaultValue={selectedPokemon.name}
                 ></input>
-                <input
+
+                <select
                     onChange={(event) =>
                         setPokemonCardData((prevData) => {
```

Comment:
I feel like we can make a helper function that updates any key: value pair in the pokemonCardData programmatically. I tihnk you did this in your React Course but I don't remember exactly what it looked like

Reply:
I am thinking something like this
`function updateCardData(event){
      setPokemonCardData(prevData => {
            return {
                  ...prevData,
                  [event.target.name]: event.taget.value
            }
      })
}`

And then we just need to give each form element a name value that matches what the API is expecting (i.e. pokemonCardSet)

Reply:
then you can use that function in every onChange handler